src/components/tools/jobs.js

```
@@ -0,0 +1,68 @@
+            progress: progress
+        })});
+    return tmpJobs.sort((a,b) => (b.date - a.date)).map(j => {
+        j.date= dateTostr(new Date(j.date));
+        return j;
+    });
+}
+
+
+export const getJobData = async (accessToken, linkJob) => {
+    const jobs = await getJobsData(accessToken);
+    return jobs.find(j => j.data && j.data.randomKey && j.data.randomKey === linkJob);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)